modules/tinymce/src/plugins/table/test/ts/browser/ui/TableDialogWithoutCssTest.ts

```diff
@@ -0,0 +1,85 @@
+import { describe, it } from '@ephox/bedrock-client';
```

Comment:
I think it would be better to add these tests to the existing TableDialogWithCssTest.ts file instead. That way we can properly check the width is correctly applied and kept for both when `table_style_css` is `false` and `true`.

Reply:
There are already tests that check this in that file. The reason this file was added was to test it with that option set to false.

Reply:
I could be missing it but I don't see any tests that check the width is correct after submitting the dialog in the existing test file.